quipucords/fingerprinter/jboss_brms.py

```diff
@@ -138,25 +127,42 @@ def detect_jboss_brms(source, facts):
     }
     product_dict = {'name': PRODUCT}
 
+    versions = set()
+    found_kie_version = False
+    for filename in kie_files:
+        category = classify_kie_file(filename)
```

Comment:
Are we able to tweak this raw fact to capture home directory so we can differentiate between multiple installs?

Reply:
Can do.